app/decorators/user_decorator.rb

```diff
@@ -97,10 +97,6 @@ def fully_banished?
     articles_count.zero? && comments_count.zero? && suspended?
   end
 
-  def stackbit_integration?
-    access_tokens.any?
```

Comment:
is this integration the only use of User's `has_many :access_tokens` doorkeeper integration (dev as an auth provider)

https://github.com/forem/forem/pull/3504 was where we added this (Mac or Ben may have more context on what this is used for)

If we're not otherwise using doorkeeper and there's no plan to use it, it might make sense to examine dropping that library as well.

Reply:
I did a bit of digging…and I think it is, but I'm _not_ touching that relationship at this time